scripts/cloudfoundry/bat-cdn/readme.md

```diff
@@ -0,0 +1,29 @@
+The Ruby script is to assist in generating the Cloudfoundry 'cf update-service <service_name>' command.
+
+As new custom domains are onboarded, there is the need to update the existing CDN service on Cloudfoundry.
+
+The CDN service for QA, Staging and Prod are bat-cdn-qa, bat-cdn-staging and bat-cdn-prod respectively.
+
+An example of the 'cf update-service <service_name>' command is described below:-
+
+cf update-service bat-cdn-qa -c \
+'{"headers":["Accept","Authorization"],"domain":"qa.find-postgraduate-teacher-training.service.gov.uk,qa.register-trainee-teachers.education.gov.uk,qa.api.publish-teacher-training-courses.service.gov.uk,qa.publish-teacher-training-courses.service.gov.uk"}'
+
+The Ruby script also includes a control file. This yaml file (cdn-config.yml) contains the parameters for each environment.
+
+Usage:
+
+In order to add a new custom domain to the CDN service, update the appropriate cdn-config.yml with correct values.
+please note, the ruby file takes single argument, which must be supplied when the script is instigated.
+The supplied argument (ARVG) represents the intended target environment. To run the script from the shell, do the following:-
+
+#QA
```

Comment:
The headings are broken in markdown: https://github.com/DFE-Digital/bat-infrastructure/blob/f94150c0ebb451a012f894a7d2ad9ced73341a6a/scripts/cloudfoundry/bat-cdn/readme.md
It should probably be: `## QA`

Reply:
done...